### ISSUE TITLE
Error: fitToParent is not a function

### ISSUE THREAD

**Issue by Xarcell** [funkhaus/fitToParent#6]
I use this script for YouTube videos in Blogspot posts. I have used it in the past and it works fine.

However, using it now, I get the error. 

**Existing Old Usage that works**: 

- jQuery 1.6
- fitToParent.js version, unknown

**Old Initialization Script**
```
 jQuery('.post-body iframe').fitToParent();
 jQuery(window).resize(function(){ jQuery('.post-body iframe').fitToParent(); });
```

**New Usage that does not work(error)**:

- jQuery 3.3.1
- fitToParent.js version, latest here on Github
- CSS flexbox used
- CSS width: calc() used on parent container

**New Initialization Script**
```
 jQuery('.youtube-video').fitToParent();
jQuery(window).resize(function(){ jQuery('.youtube-video').fitToParent(); });
```
If I use the old fitToParent.js with the new usage, it partially works. The iframe is resized, larger, but not 100% width of parent.

I can post the old fitToParent.js at request, it wasn't included to save space.

Iframes with the class "youtube-video" exists, and is visible in the DOM.

**Comment by drewbaker** [comment]
Yeah I think it's actually due to the Flexbox usage. I've found that jQuery doesn't know how to measure the height of a Flexbox sometimes. 

We use this now instead of fitToParent: https://github.com/fregante/intrinsic-scale

It's just a math function, you tell it how big the parent is, then it tells you what the iFrame should be to fit in it. It's very lightweight and works in server side rendered environments like Vue/Nuxt.
